code/headers/lidar.hpp

```diff
@@ -0,0 +1,118 @@
+#pragma once
+
+#include <lidar_packets.hpp>
+#include <usart_connection.hpp>
+
+namespace r2d2 {
+    namespace distance {
```

Comment:
maybe consider namespace chaining this is a bit less typing and error prone, `namespace r2d2::distance { `is sufficient